src/devices/machine/upd765.cpp

```diff
@@ -679,8 +679,8 @@ uint8_t upd765_family_device::fifo_pop(bool internal)
 	memmove(fifo, fifo+1, fifo_pos);
 	if(!fifo_write && !fifo_pos)
 		disable_transfer();
-	int thr = fifocfg & 15;
-	if(fifo_write && fifo_expected && (fifo_pos <= thr || (fifocfg & 0x20)))
+	int thr = fifocfg & FIF_THR;
+	if(fifo_write && fifo_expected && (fifo_pos <= thr || (fifocfg & FIF_DIS)))
```

Comment:
Just cleaning up some hardcoded values since the enum values were used in `fifo_push`:
```
FIF_THR  = 0x0f,
FIF_DIS  = 0x20,
```